Formula/qrcp.rb

```diff
@@ -0,0 +1,42 @@
+class Qrcp < Formula
+  desc "Transfer files to and from your computer by scanning a QR code"
+  homepage "https://claudiodangelis.com/qrcp"
+  url "https://github.com/claudiodangelis/qrcp/archive/0.6.3.tar.gz"
+  sha256 "2d39ba661aad9c60b816bc06f53ef4f3d8747e11d5fc27c104d3687d1e77204c"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    config_file = testpath/"config.txt"
+    test_file = testpath/"test.txt"
+    test_file_content = "Hello there, big world"
+    test_file.write test_file_content
+    port = free_port
+    server_url = "http://localhost:#{port}/send/testing"
+
+    config_file.write <<-EOS
```

Comment:
```suggestion
    (testpath/"config.txt").write <<-EOS
```

Reply:
Again, this would also need to be hard coded on the exec call, like 
```
exec bin/"qrcp", "-c", (testpath/"config.txt", "--path", "testing", (testpath/"test.txt")
```

Is this preferrable?

Reply:
Typing it twice is preferable (and it wouldn't need brackets in the test call)